paper.tex

```diff
@@ -102,106 +102,101 @@ \subsection{Data exchange mechanism for IoT}
 
 \subsection{Trusted IoT Trading Infrastructure}
-Multiple infrastructures for trusted IoT data trading have been proposed. Paolo Missier et. al\cite{MindMyValue} presented an IoT brokered infrastructure based marketplace which enables trading with Ethereum smart contracts. The brokers are only responsible for data transmission in order to adapt flexible agreement between participants. Unlike the standard pub/sub model that matches publishers and subscribers that participants are unaware of each other. However, the data cubes (i.e., a tuple of information, such as provider, subscriber and time period) are stored in a centralized Cassandra NoSQL database, referring as TrackerDB, is guarateed to be tamperproof but the risk of single point failure still exists. 
+Multiple infrastructures for trusted IoT data trading have been proposed. Paolo Missier et. al\cite{MindMyValue} presented an IoT brokered infrastructure based marketplace which enables trading with Ethereum smart contracts. The brokers are only responsible for data transmission in order to adapt flexible agreement between participants. Unlike the standard pub/sub model that matches publishers and subscribers that participants are unaware of each other. However, the data cubes (i.e., a tuple of information, such as provider, subscriber and time period) are stored in a centralized Cassandra NoSQL database, referring as TrackerDB, is guarateed to be tamperproof but the risk of single point failure still exists.
```

Comment:
Shall we explain "TrackerDB"?

Reply:
I don't think so. 
Would it be less misunderstanding if we just mention a centralized database is used here and remove the term (TrackerDB) they use?